frontend/websites/puconvocation-com/src/app/[locale]/(console)/console/attendees/page.tsx

```diff
@@ -10,15 +10,167 @@
  * treaties. Unauthorized copying or distribution of this software
  * is a violation of these laws and could result in severe penalties.
  */
+"use client";
 
-import { JSX } from "react";
+import { Fragment, JSX, useState } from "react";
+import { useQuery } from "@tanstack/react-query";
+import { Button } from "@components/ui";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@components/ui/card";
+import { AttendeeService } from "@services/index";
+import { StatusCode } from "@enums/StatusCode";
+import {
+  UsersIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from "@heroicons/react/24/solid";
+import { ProgressBar } from "@components/index";
+
+const attendeeService = new AttendeeService();
 
 export default function AttendeePage(): JSX.Element {
+  const [showAttendees, setShowAttendees] = useState(false);
+  const [page, setPage] = useState(0);
+  const [attendeeCount, setAttendeeCount] = useState(0);
+
+  const {
+    data: attendees,
+    isLoading: isAttendeeLoading,
+    isError: attendeeError,
+  } = useQuery({
+    queryKey: ["homeAttendeesList", page],
+    refetchOnWindowFocus: false,
+    queryFn: async () => {
+      const response = await attendeeService.getAllAttendees(page, 10);
+      if (
+        response.statusCode === StatusCode.SUCCESS &&
+        "payload" in response &&
+        typeof response.payload === "object"
+      ) {
+        setAttendeeCount(response.payload.totalCount);
+        return response.payload.attendees;
+      }
+      return [];
+    },
+  });
+
+  const currentAttendees = attendees || [];
+
   return (
-    <div className={"flex min-h-screen"}>
-      <div className={"m-auto"}>
-        <div>
-          <h1 className={"text-2xl font-bold"}>Attendees</h1>
+    <div className={"flex min-h-screen flex-col space-y-10 p-4 md:p-10"}>
+      <div className={"space-y-3"}>
+        <h1 className={"flex items-center text-2xl font-bold"}>
+          <UsersIcon className="mr-2 h-6 w-6 text-red-600" /> Attendees
+        </h1>
+        <p className={"text-xs text-gray-600"}>
+          View and manage the complete list of attendees for the convocation.
+        </p>
+      </div>
+
+      <div className={"flex min-h-screen flex-col items-center p-1"}>
+        <div className="mb-2 grid w-full grid-cols-1 gap-6 lg:grid-cols-1">
+          {/* Total Attendees */}
+          <Card className="h-fit w-full border border-gray-300 p-4">
+            <CardHeader>
+              <CardTitle>Total Attendees</CardTitle>
+              <CardDescription>
+                Showing total number of attendees for the convocation.
+              </CardDescription>
+            </CardHeader>
+            <CardContent className="flex items-center justify-between">
+              <span className="text-lg">{attendeeCount}</span>
+              <Button
+                onClick={() => setShowAttendees(!showAttendees)}
```

Comment:
Here we need to make a HTTP request to backend in order to actually lock the attendees. The method to do this is already defined  in `AttendeeController.tsx` (previously known as `AttendeeService.tsx`) with this  [method](https://github.com/mihirpaldhikar/pu-convocation/blob/1da36c122be2e5c797c859404ce40277d26357ed/frontend/websites/puconvocation-com/src/controllers/AttendeeController.ts#L78)

Also. the value of this should be obtained from [`useRemoteConfig`](https://github.com/mihirpaldhikar/pu-convocation/blob/1da36c122be2e5c797c859404ce40277d26357ed/frontend/websites/puconvocation-com/src/hooks/useRemoteConfig.ts#L17) hook. Specifically, you need to use [this property](https://github.com/mihirpaldhikar/pu-convocation/blob/1da36c122be2e5c797c859404ce40277d26357ed/frontend/websites/puconvocation-com/src/dto/RemoteConfig.ts#L17). 

When a request to change the lock status is made, the global state of the `RemoteConfig` should also be updated with `dispatcher`.